src_py/grndwork_api_client/dtos.py

```diff
@@ -0,0 +1,72 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, TypedDict
+
```

Comment:
Add RefreshToken? and use instead of untyped dict?
same with PostDataPayload

Reply:
Done!